Tests/Grid/Mapping/Metadata/DriverHeapTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace APY\DataGridBundle\Grid\Tests\Mapping\Metadata;
+
+use APY\DataGridBundle\Grid\Mapping\Metadata\DriverHeap;
+use APY\DataGridBundle\Grid\Mapping\Source;
+use PHPUnit\Framework\TestCase;
+
+class DriverHeapTest extends TestCase
+{
+    public function testCompareOk()
+    {
+        $priority = rand(0, 100);
+
+        $driverHeap = new DriverHeap();
+
+        $this->assertEquals(0, $driverHeap->compare($priority, $priority));
+    }
+
+    public function testPriority1MoreThanPriority2()
+    {
+        $priority1 = rand(100, 1000);
+        $priority2 = rand(0, 100);
```

Comment:
There is not need to use `random`: use fixed number instead. Moreover `priority2` can be equal to `priority1` as `rand` is inclusive.